robocopkaka/reminder_app
# Ft add reminders resource

#### What does this PR do?
* Allow users add reminders
#### Description of Task to be completed?
* Add reminders resource
#### How should this be manually tested?
* Start your server
* Go to `http://localhost:3000/reminders/new`
* Enter the `title` and `description`
* Pick the days you'd want to be reminded on
* Enter the time
* Click on `Create reminder`
* You should be redirected to the home page if it was successful or see error messages on the form if it wasn't
#### Any background context you want to provide?
N/A
